### ISSUE TITLE
IDMerger fails in TOPPAS

### ISSUE THREAD

**Issue by lars20070** [OpenMS/OpenMS#2166]
[This](https://gist.github.com/lars20070/b97e58533f36fef5ae1346c1459b2a60) IDMerger workflow fails for any `idXML` [files](https://gist.github.com/lars20070/f03d5c77ac65a7022e82b9d1fe390884) with
```
Did not find upstream nodes with un-recycled names. Something is fishy!

==============================================================================
11:53:57 ERROR: IDMerger failed!
Did not find upstream nodes with un-recycled names. Something is fishy!
```
From the command line `IDMerger` works fine.
<img width="441" alt="screen shot 2016-09-02 at 11 53 34" src="https://cloud.githubusercontent.com/assets/6149805/18200241/2477c172-7104-11e6-9c43-ac2ae9c2beae.png">

**Comment by timosachsenberg** [comment]
@cbielow should this work?

**Comment by timosachsenberg** [comment]
error triggered in: https://github.com/OpenMS/OpenMS/blob/9f736a67f5ac09b61c79d3841d1b0dfe12c49d65/src/openms_gui/source/VISUAL/TOPPASToolVertex.cpp#L929

**Comment by timosachsenberg** [comment]
potentially introduced in https://github.com/OpenMS/OpenMS/commit/e751ff4b50699c97f263f872122fcf3e3c212297

**Comment by lars20070** [comment]
Replacing `TOPPAS.exe` by the `TOPPAS` executable from OpenMS 2.0 does not resolve the issue.
Same error message.

**Comment by lars20070** [comment]
Resolved in #2180.
